pkg/repositories/models/schedulable_entity.go

```diff
@@ -0,0 +1,31 @@
+package models
+
+import (
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+	"time"
+)
+
+// Database model to encapsulate metadata associated with a SchedulableEntity
+type SchedulableEntity struct {
+	BaseModel
+	SchedulableEntityKey
+	CronExpression      string
+	FixedRateValue		uint32
+	Unit                admin.FixedRateUnit
+	LastExecutionTime   *time.Time
+	KickoffTimeInputArg string
+	Active              bool
```

Comment:
i think you want this to be a pointer so gorm actually updates the model when you deactivate a schedule, see https://gorm.io/docs/create.html#Default-Values

Reply:
Thanks for pointing that out